tests/unit/test_tutoring_model.py

```diff
@@ -61,3 +61,46 @@ def test_get_coordinates(
     assert result["KÜ"] == (13, 13, 13, 13)
     assert result["EK"] == (12, 12, 12, 12)
     assert result["LZ"] == (-12, -12, -12, -12)
+
+
+def test_prepare_les_for_ga():
```

Comment:
Could you add more tests for when the creation is failing. We need to cover all the branches in the code. Thus making multiple tests. :)